static/styles/settings.css

```diff
@@ -2040,3 +2029,21 @@ input[type="checkbox"] {
         margin-top: 10px;
     }
 }
+
+.settings_panel_list_header {
+    position: relative;
+    padding-bottom: 10px;
+
+    .list_title {
+        display: inline-block;
+        font-size: 1.3em;
+        line-height: 1.3;
+        margin-top: 6px;
+    }
```

Comment:
What's the thinking of using sizing CSS here vs. the common CSS we'd get by preserving the previous `h3` tags? We have some CSS related to h3s in settings, which possibly can be deleted if we prefer this approach.

Reply:
The main problem here is that we're trying to make sure that the title on the left mostly vertically aligns with filter box on the right, as well as having roughly the same dimensions.  You have to do fiddly stuff like the `margin-top` to make it look nice, and I suspect a designer could do an iteration on this to make it look nicer.

I really don't want to couple this problem to all the other h3 tags in our system.

Reply:
We basically only have H3 tags in settings. The main reason I mention this is that we have special CSS for e.g. `help_settings_link` for H3 tags in settings pages.

Reply:
Yeah, and this is in fact a bug -- the "Deactivated users" heading has a blue link because that CSS is no longer applying.

I think it'd be better to fix by preserving the h3s; that's semantically the role of these elements; almost all of our H3s in Zulip are in the settings system, the CSS is scoped to settings pages, and these H3s are primarily for the exact section heading purpose we have here.

Reply:
I went to H4 tags because H3 tags look too large to me.